packages/connectivity_plus/connectivity_plus_platform_interface/lib/src/enums.dart

```diff
@@ -16,5 +16,10 @@ enum ConnectivityResult {
   none,
 
   /// VPN: Device connected to a VPN
-  vpn
+  ///
+  /// Note that on iOS and macOS comes [other], not [vpn]
```

Comment:
Could you write an explanation what this exactly means? In case of vpn it returns other? Only in a simulator, or on any device?

Reply:
I don't quite understand how to describe it correctly. Yes in case of vpn it returns other, on any device.

[Documentation describing type other](https://developer.apple.com/documentation/network/nwinterface/interfacetype/other).

Reply:
Would that be more correct?

`Note for iOS and macOS: in the case of [vpn] it returns [other], both in the simulator and on any device.`